parsley-debug/shared/src/main/scala/parsley/debugger/internal/ToXMap.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Parsley Contributors <https://github.com/j-mie6/Parsley/graphs/contributors>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+package parsley.debugger.internal
+
+trait ToXMap[M[_, _]] {
```

Comment:
public exposed API in internal, should be `private [internal]`!

Reply:
(same with object below)

Reply:
Removed completely, it turns out it wasn't needed.